bookshelf/bookshelf.d.ts

```diff
@@ -177,8 +191,12 @@ declare module 'bookshelf' {
 			keys() : string[];
 			last() : T;
 			lastIndexOf(value : any, fromIndex? : number) : number;
-			map(predicate? : Lodash.ListIterator<T, boolean>|Lodash.DictionaryIterator<T, boolean>|string, thisArg? : any) : T[];
+
+			// See https://github.com/Microsoft/TypeScript/blob/v1.8.10/lib/lib.core.es7.d.ts#L1122
```

Comment:
You should probably reference the lodash docs if you want to reference something: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/lodash/lodash-3.10.d.ts#L7119 or https://github.com/lodash/lodash/blob/3.10.1/lodash.src.js#L6864